README.md

```diff
@@ -1,126 +1,258 @@
+Requirements to build:
+- C++17 compiler and standard library
+- boost 1.67+
+- CMake 3.8+
+- LLVM 7 - 11 - for Linux only
+  - newer versions do not work
+- openssl 1.1+
+- libcurl
+- curl
+- libusb
+- git
+- GMP
+- Python 3
+- zlib
 
-#### Building Pinned Build Binary Packages
-In the directory `<leap src>/scripts` you will find the two scripts `install_deps.sh` and `pinned_build.sh`. If you haven't installed build dependencies then run `install_deps.sh`. Then run `pinned_build.sh <dependencies directory> <leap build directory> <number of jobs>`.
+### Step 1. Clone the Leap repository
 
-The dependencies directory is where the script will pull the C++ dependencies that need to be built with the pinned compiler for building the pinned binaries for binary packaging.
+If you don't have the Leap repo cloned to your computer yet, [open a terminal](https://itsfoss.com/open-terminal-ubuntu) and navigate to the folder where you want to clone the Leap repository:
+```bash
+cd ~/Downloads
+```
+Clone Leap using either HTTPS:
+```bash
+git clone --recursive https://github.com/AntelopeIO/leap.git
+```
+or SSH:
+```bash
+git clone --recursive git@github.com:AntelopeIO/leap.git
+```
 
-The binary package will be produced in the Leap build directory that was supplied.
+> ℹ️ **HTTPS or SSH git clone** ℹ️  
+Both an HTTPS or SSH git clone will yield the same result: a folder named `leap` containg our source. It doesn't matter which type you use (unless you need to push to remote).
 
-#### Manual (non "pinned") Build Instructions
+Navigate into that folder:
+```bash
+cd leap
+```
 
-<details>
-  <summary>Ubuntu 20.04 & 22.04 Build Instructions</summary>
+### Step 2. Select release tag or branch
 
-Install required dependencies: 
+Choose which [release tag](https://github.com/AntelopeIO/leap/releases) or [branch](#branches) you would like to build, then check it out. If you are not sure, use the [latest release](https://github.com/AntelopeIO/leap/releases/latest). For example, if you want to build release 3.1.2 then you would check it out using its tag, `v3.1.2`. In the example below, replace `v0.0.0` with your selected release tag accordingly:
+```bash
+git fetch --all --tags
+git checkout v0.0.0
 ```
-apt-get update && apt-get install   \
-        build-essential             \
-        cmake                       \
-        curl                        \
-        git                         \
-        libboost-all-dev            \
-        libcurl4-openssl-dev        \
-        libgmp-dev                  \
-        libssl-dev                  \
-        libusb-1.0-0-dev            \
-        llvm-11-dev                 \
-        pkg-config
+
+Once you are on the branch or release tag you want to build, make sure everything is up-to-date:
+```bash
+git pull
+git submodule update --init --recursive
+```
+
+#### Jobs Flag
+⚠️ **A Warning On Parallel Compilation Jobs (`-j` flag)** ⚠️  
```

Comment:
Might want to make this a quote too by prepending this line with ">". That way we don't distract the user from performing the main task which is building and installing Leap.

Reply:
Done.